apps/fabric-website/src/pages/GetStarted/GetStartedPage.tsx

```diff
@@ -173,30 +173,35 @@ export class GetStartedPage extends React.Component<any, any> {
 
         <div className={styles.instructionsSection}>
           <h2 id="react">Get started with Fabric React</h2>
-          <p>
-            Use NPM to get Fabric components and core styling. All you need is{' '}
-            <a className={styles.getStartedLink} href="https://nodejs.org/en/">
-              node.js
-            </a>{' '}
-            and{' '}
-            <a className={styles.getStartedLink} href="http://gulpjs.com/">
-              gulp
-            </a>
-            .
-          </p>
+          <p>All you need to get started with Fabric is Node.js and npm.</p>
 
           <ol className={styles.steps}>
             <li>
-              <p>To install the Fabric React NPM package, from the root of your project, run:</p>
+              <p>
+                Install Node.js (at least 8.x) from the
+                <a href="https://nodejs.org">Node.js website</a>.
```

Comment:
space between "the" and anchor is getting clipped. You can just do something like

```tsx
<p>
  { "Install Node.js (LTS version) from the " }
  <a ... />
  { "." }
</p>